homeassistant/components/cast/media_player.py

```diff
@@ -461,55 +456,26 @@ def media_seek(self, position):
         media_controller = self._media_controller()
         media_controller.seek(position)
 
-    async def async_browse_media(self, media_content_type=None, media_content_id=None):
-        """Implement the websocket media browsing helper."""
-        kwargs = {}
+    async def _async_root_payload(self, content_filter):
+        """Generate root node."""
         children = []
+        # Add media browsers
+        for _, platform in self.hass.data[CAST_DOMAIN].items():
+            if not hasattr(platform, "async_get_media_browser_root_object"):
+                continue
```

Comment:
This should happen inside `_process_cast_platform`

Reply:
Fixed in 10ffd80